packages/manager/modules/nasha/src/components/task-tracker/task-tracker.utils.js

```diff
@@ -25,6 +26,8 @@ export const createTaskTrackerStateOptions = (queryParams = []) => ({
         .filter((key) => queryParams.includes(key))
         .reduce((set, key) => ({ ...set, [key]: params[key] }), {});
     },
+    trackingData: /* @ngInject */ ($transition$) =>
+      $transition$.params().trackingData,
```

Comment:
**question:** If state param is not put in the URL, it can be lost after refresh. So is this case taken care?

Reply:
If there is a refresh of the page, there is no tracking in this case (and no error).